packages/@aws-cdk/aws-cloud9/README.md

```diff
@@ -104,3 +104,39 @@ new cloud9.Ec2Environment(this, 'C9Env', {
   imageId: cloud9.ImageId.AMAZON_LINUX_2,
 });
 ```
+
+## Specifying Owners
+`Owner` is a user that owns a Cloud9 environment . `Owner` has their own access permissions, resources. And we can specify an `Owner`in an Ec2 environment which could be of two types, 1. AccountRoot and 2. Iam User. It allows AWS to determine who has permissions to manage the environment, either an IAM user or the account root user
+
+### AccountRoot
+
+```ts
+new cloud9.Ec2Environment(this, 'C9Env', {
+  // provides root account id.
+  owner: cloud9.Owner.AccountRoot('root account id')
+})
+```
+
+### Iam User
+
+```ts
+import * as iam from '@aws-cdk/aws-iam';
+
+const user = new iam.User(stack, 'User');
+// provides an iam user.
+new cloud9.Ec2Environment(this, 'C9Env', {
+  owner: cloud9.Owner.User(user)
+})
+```
+
+### create a new Cloud9 environment with an owner as an Iam User.
+
+```ts
+const user = new iam.User(stack, 'User');
+declare const vpc: ec2.Vpc;
+new cloud9.Ec2Environment(this, 'C9Env', {
+  vpc,
+  imageId: cloud9.ImageId.AMAZON_LINUX_2,
+  owner: cloud9.Owner.User(user)
+});
+```
```

Comment:
This example isn't necessary any more, the previous example already covers that.